Dockerfile

```diff
@@ -18,7 +18,7 @@ RUN apt-get update && apt-get install -y \
 # System-level PHP setup
 RUN add-apt-repository ppa:ondrej/php -y
 RUN apt-get install -y \
-    php7.4 php7.4-curl php7.4-fpm php7.4-gd php7.4-mbstring php7.4-mysql php7.4-xml
+    php7.4 php7.4-curl php7.4-fpm php7.4-gd php7.4-intl php7.4-mbstring php7.4-mysql php7.4-xml
```

Comment:
Added the "server" label because of the new dependency.

Reply:
I'm thinking we should start potentially creating "deployment" tasks with an easy list for Jani of stuff that needs doing/changing since the last one.

Reply:
#1391